src/main/include/subsystems/LimeLightUtil.h

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include <frc/controller/PIDController.h>
+#include "LimeLightHelpers.h"
+#include "SwerveDrive.h"
+#include "Constants.h"
+
+namespace t34
+{
+
+    class LimeLightUtil
+    {
+
+    public:
+
+        enum class TargetMode
+        {
+            kSpeaker,
+            kAmp,
+            kTrap
+        };
+
+    private:
+
+        frc::PIDController m_ll_swerve_pid;
```

Comment:
May I be a stickler about spelling things out?

```suggestion
        frc::PIDController m_limelight_swerve_pid;
```